modules/SofaEditor/python/tests/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(PythonSofaEditor_test)
+
+####################################################################################################
+### Module dependencies
+####################################################################################################
+find_package(SofaPython 2.7 QUIET)
+if(!SofaPython_FOUND)
+    message(FATAL_ERROR "-- 'PythonSofaEditor_test' is disabled because 'SofaPython' is missing or not activated.")
+    return()
```

Comment:
equivalent to `find_package(SofaPython 2.7 REQUIRED)` ?

Reply:
I don't think so. 
REQUIRED will make the cmake process to fail while the pattern I'm using is just not doing the python binding.

Reply:
`message(FATAL_ERROR` will make the CMake process fail